recipes/enable-postfix-smarthost.rb

```diff
@@ -44,13 +44,14 @@
 end
 
 execute 'tell postfix where CA cert is' do
-  command %Q|postconf -e 'smtp_tls_CAfile = /etc/ssl/certs/ca-certificates.crt'|
+  command %Q|postconf -e 'smtp_tls_CAfile = /etc/ssl/certs/ca-bundle.crt'|
   retries 5
   retry_delay 5
 end
 
 service 'postfix' do
-  action :reload
+  supports :reload => true
+  action [:reload, :start]
```

Comment:
postfix service doesn't start automatically as it did in ubuntu.

also, the chef recipe will exec both of those actions sequentialy, so if the service is already running you'll get this:
```
[2022-01-21T14:27:06-05:00] INFO: Processing service[postfix] action reload (oc-opsworks-recipes::enable-postfix-smarthost line 52)
[2022-01-21T14:27:06-05:00] DEBUG: service[postfix] supports status, running
[2022-01-21T14:27:06-05:00] DEBUG: service[postfix] is running
[2022-01-21T14:27:08-05:00] INFO: service[postfix] reloaded
[2022-01-21T14:27:08-05:00] INFO: Processing service[postfix] action start (oc-opsworks-recipes::enable-postfix-smarthost line 52)
[2022-01-21T14:27:08-05:00] DEBUG: service[postfix] supports status, running
[2022-01-21T14:27:08-05:00] DEBUG: service[postfix] is running
[2022-01-21T14:27:08-05:00] DEBUG: service[postfix] already running - nothing to do
```
```

Reply:
Looks ok to me.